CRM/Core/BAO/CustomValueTable.php

```diff
@@ -550,7 +550,13 @@ public static function setValues(&$params) {
           ));
         }
 
+        $entity = $fieldInfo['custom_group']['extends'];
+        $contactTypes = CRM_Contact_BAO_ContactType::basicTypes(TRUE);
+        if (in_array($entity, $contactTypes)) {
+          $entity = 'Contact';
+        }
```

Comment:
@seamuslee001 there's already a function for this:

```suggestion
        $entity = CRM_Core_BAO_CustomGroup::getEntityFromExtends($fieldInfo['custom_group']['extends']);
```

Reply:
Thanks @colemanw updated to it now